test/object-store/migrations.cpp

```diff
@@ -2399,9 +2576,10 @@ TEST_CASE("migration: Additive") {
         realm2->commit_transaction();
 
         REQUIRE_NOTHROW(realm->refresh());
-        REQUIRE(realm->schema() == schema);
+        REQUIRE(realm->schema() != schema);
```

Comment:
Can you strengthen this validation a bit? Eg. in addition to having the != check, you can add `realm->schema().compare(schema)` and make sure that there is only one additional property added in the difference.

Reply:
done